### PR TITLE
use variant-filled-primary class for active nav item to also use corr…

### PR DESCRIPTION
**Pull request by vhochstein**
currently example to highlight active nav item is using a bg background color class, however, that might lead to "incompatible" textcolor usage.

Therefore I would suggest to use a variant-filled-primary class instead, which will automatically select correct textcolor as well.

This is my first PR, so not sure what I have to comply to, but since it is an quite easy simple change.. 

## Linked Issue
None

## Description
currently example to highlight active nav item is using a bg background color class, however, that might lead to "incompatible" textcolor usage.

Therefore I would suggest to use a variant-filled-primary class instead, which will automatically select correct textcolor as well.

## Checklist

Please read and apply all [contribution requirements](https://www.skeleton.dev/docs/contributing).

- [x] This PR targets the `dev` branch (NEVER `master`)
- [x] Documentation reflects all relevant changes
- [x] Branch is prefixed with: `docs/`, `feat/`, `chore/`, `bugfix/`
- [x] Ensure Svelte and Typescript linting is current - run `pnpm ci:check`
- [x] Ensure Prettier linting is current - run `pnpm format`
- [x] All test cases are passing - run `pnpm test`
- [x] Includes a changeset (if relevant; see above)

### CODE REVIEW

**Comment by Sarenor** [comment]
That's not a bad improvement!
Usually, we like some discussion in an issue or Discord first and here's the contribution guide: https://www.skeleton.dev/docs/contributing
Only thing that's not quite correct is that the branch isn't prefixed with docs/ as far as I can see (and the pr out of nowhere no discussion).
Since it's a good suggestion I'd accept it though, will wait for @endigo9740 s final approval though.

**Review by Sarenor — APPROVED** [review]

**Comment by endigo9740** [comment]
@vhochstein seems like a good change. Like @Sarenor said, we typically expect some upfront communication, but given the small scale of this we'll let it through. Please also make sure to use the checklist provided in the PR template as well. I've handled this for you this time. Thanks for the contribution.

**Pull request merged**
